src/generate.py

```diff
@@ -30,6 +33,19 @@
     with config.open(encoding='utf8') as f:
         cfg = yaml.safe_load(f)
 
-    bank = load_data(cfg)
+    # where .cache and other artifacts are located
+    artifacts_path = None
+    if args.artifacts:
+        artifacts_path = Path(args.artifacts)
+        if not artifacts_path.exists():
+            print(f'Artifacts path not found: {artifacts_path}')
+            exit()
```

Comment:
`exit(f'Artifacts path not found: {artifacts_path}')` is better because it returns non 0 exit code